src/main/cpp/chassis/headingStates/FaceCenterStage.cpp

```diff
@@ -31,7 +31,7 @@ FaceCenterStage::FaceCenterStage() : FaceTarget(ChassisOptionEnums::HeadingOptio
 
 std::optional<frc::Pose3d> FaceCenterStage::GetAprilTagPose()
 {
-    int aprilTag = (FMSData::GetInstance()->GetAllianceColor() == frc::DriverStation::kBlue ? FaceTarget::BLUE_AMP : FaceTarget::RED_AMP);
+    int aprilTag = (FMSData::GetInstance()->GetAllianceColor() == frc::DriverStation::kBlue ? FaceTarget::RED_STAGE_CENTER : FaceTarget::BLUE_STAGE_CENTER);
```

Comment:
The order should be reversed the format of this statement is if condition ? true : false, so you are saying if it is blue use the red stage